git_machete/docs.py

```diff
@@ -142,7 +163,7 @@
           <b>-b, --branch=<branch></b>      Branch to set the annotation for.
           <b>-H, --sync-github-prs</b>      Annotate with GitHub PR numbers and authors where applicable.
     """,
-    "clean": """
+    "clean": f"""
         <b>Usage:
```

Comment:
Oh! here's the problem:

![image](https://user-images.githubusercontent.com/3383210/168443702-416ec94e-1e0f-4c07-ab8c-412fb85af11e.png)

Unclosed tag

Reply:
Great catch, thx